### PR TITLE
[doc] Update README.md

### PR DESCRIPTION
**Pull request by Jamie-VESoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,42 +22,51 @@
...
-* Highly scalable
-* Fault tolerant
-* Strong data consistency
-* SQL-like query language
-* Role-based access control
-* Support multiple storage engines
+* Storage and computing separation
+* Horizontal scalability
+* Strong data consistency per RAFT protocol
+* SQL-like query language with OpenCypher interface
```
Comment (jude-zhu) [inline README.md]: ”OpenCyper interface“ not at this moment  😹😹

File: README.md [hunk 2]
```diff
@@ -20,42 +22,50 @@
...
 * Symmetrically distributed
-* Highly scalable
-* Fault tolerant
-* Strong data consistency
-* SQL-like query language
-* Role-based access control
-* Support multiple storage engines
+* Storage and computing separation
+* Horizontal scalability
+* Strong data consistency per RAFT protocol
```
Comment (whitewum) [inline README.md]: by

File: README.md [hunk 3]
```diff
@@ -20,42 +22,50 @@
...
+* SQL-like query language with OpenCypher interface
```
Comment (whitewum) [inline README.md]: SQL-like graph query language

File: README.md [hunk 4]
```diff
@@ -20,42 +22,50 @@
...
+* Role-based access control for higher level security
+
+## Roadmap
+
+See our [Roadmap](https://github.com/vesoft-inc/nebula/wiki/Nebula-Graph-Roadmap-RC3) for what's coming soon in Nebula Graph.
```
Comment (whitewum) [inline README.md]: https://github.com/vesoft-inc/nebula/wiki#roadmap

File: README.md [hunk 5]
```diff
@@ -20,42 +22,50 @@
...
+
+## Quick start
 
-## How Can I Get Nebula Graph
+There are two ways you can quickly start Nebula Graph:
+* By [installing source code](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md)
+* By [docker image](https://hub.docker.com/r/vesoft/nebula-graph)
```
Comment (whitewum) [inline README.md]: https://github.com/vesoft-inc/nebula-docker-compose

File: README.md [hunk 6]
```diff
@@ -20,42 +22,50 @@
...
+* Role-based access control for higher level security
+
+## Roadmap
+
+See our [Roadmap](https://github.com/vesoft-inc/nebula/wiki/Nebula-Graph-Roadmap-RC3) for what's coming soon in Nebula Graph.
+
+## Quick start
 
-## How Can I Get Nebula Graph
+There are two ways you can quickly start Nebula Graph:
```
Comment (whitewum) [inline README.md]: @wilsonyou please help to re-write this sub-section.

File: README.md [hunk 7]
```diff
@@ -20,42 +22,50 @@
...
-# Nebula Graph
-
-**Nebula Graph** is the only solution in the world capable of hosting graphs with dozens of billions of vertices (nodes) and trillions of edges, while still only having milliseconds of latency.
+# What is Nebula Graph?
 
-**Nebula Graph's** goal is to provide reading, writing, and computing with high concurrency and low latency for super large scale graphs. **Nebula Graph** is an open source project and we are looking forward to working with the community to popularize and promote the graph database.
+**Nebula Graph** is an open-source graph database capable of hosting super large scale graphs with dozens of billions of vertices (nodes) and trillions of edges, with milliseconds of latency.
 
-As a graph database, **Nebula Graph** has these features:
+Compared with other graph database solutions, Nebula Graph has the following advantages:
```
Comment (Amber1990Zhang) [inline README.md]: Please bold all the **Nebula Graph**.

File: README.md [hunk 8]
```diff
@@ -20,42 +22,50 @@
...
 
 * Symmetrically distributed
```
Comment (Amber1990Zhang) [inline README.md]: Please change the CN readme.

File: README.md [hunk 9]
```diff
@@ -65,5 +75,6 @@ In order to prevent cloud providers monetizing from the project without contribu
 
 ## Contact
 
-* Please use [GitHub issue tracker](https://github.com/vesoft-inc/nebula/issues) for filling bugs or feature requests.
-* Join [![](https://img.shields.io/badge/slack-nebula-519dd9.svg)](https://nebulagraph.slack.com/archives/DJQC9P0H5/p1557815158000200).
+* Twitter:<a href="https://twitter.com/NebulaGraph">@NebulaGraph</a> 
+* <a href="https://www.linkedin.com/company/vesoft-nebula-graph/">LinkedIn page</a>
+* <a href="https://nebulagraph.slack.com/archives/DJQC9P0H5/p1557815158000200">Slack channel</a>
```
Comment (Amber1990Zhang) [inline README.md]: Please add email here.

File: README-CN.md [hunk 10]
```diff
@@ -17,39 +17,51 @@
   </a>
 </p>
 
-# Nebula Graph
+# Nebula Graph是什么？
 
-**Nebula Graph** 的目标是为超大规模图数据提供高并发，低延时的读、写及计算。**Nebula Graph** 是一个开源项目，我们期待与社区合作共同推广图数据库。
+**Nebula Graph** 是一款开源的图数据库，擅长处理亿万条边的超大规模数据集。
```
Comment (Amber1990Zhang) [inline README-CN.md]: 百亿节点万亿条边

File: README-CN.md [hunk 11]
```diff
@@ -17,39 +17,51 @@
...
-**Nebula Graph** 的主要特点：
+与其他图数据库产品相比，**Nebula Graph** 具有如下优势：
 
 * 全对称分布式架构
-* 可扩展
-* 高可用
-* 数据强一致
+* 存储与计算分离
+* 水平可扩展性
+* RAFT协议下的数据强一致
```
Comment (Amber1990Zhang) [inline README-CN.md]: RAFT 协议，中英文空格

File: README-CN.md [hunk 12]
```diff
@@ -17,39 +17,49 @@
...
-您也可以观看[视频](https://space.bilibili.com/472621355)开始学习 **Nebula Graph**。
+在开始使用 **Nebula Graph** 之前，必须通过[编译源码](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md)或者 [docker compose](https://github.com/vesoft-inc/nebula-docker-compose) 方式安装 **Nebula Graph**。您也可以观看[视频](https://space.bilibili.com/472621355)学习如何安装 **Nebula Graph**。
 
-## 参与 Nebula Graph
+## 获取帮助
 
-**Nebula Graph** 是一个完全开源的项目，欢迎开源爱好者参与 **Nebula Graph** 社区，目前有以下贡献方式：
+在使用 **Nebula Graph** 过程中遇到任何问题，都可以通过下面的方式寻求帮助：
+* [知乎](https://www.zhihu.com/org/nebulagraph/activities)
+* [SegmentFault](https://segmentfault.com/t/nebula)
```
Comment (whitewum) [inline README-CN.md]: 或者直接在GitHub上提issue

**Pull request merged**